### PR TITLE
JS shows empty axes when a plot has no data

### PR DESCRIPTION
**Pull request by TimKDJ**
So now the following will show an empty plot automatically:
```
createJaspPlot(title = "Some plot title", width = 300, height = 300)
```

### CODE REVIEW

**Comment by TimKDJ** [comment]
@vandenman I guess we could also extend this to tables:
If an empty table is generated, automatically add 1 row of dots.
I find myself using $setExpectedRows() twice per table function, once if not ready or error (`$setExpectedRows(1)`) and once if actually filling the table (`$setExpectedRows(nRows)`).

**Comment by vandenman** [comment]
Extending this to tables makes sense, but I'd rather see that changed on the cpp side. For example, when the results get send by jaspResults, if any table has 0 rows, add 1 row with ".". 

Oh and if the table gets filled quickly, do you really need the `$setExpectedRows(nRows)`?

**Comment by TimKDJ** [comment]
Why do you want this in cpp?

**Comment by vandenman** [comment]
Let me rephrase, I just don't want it in R. In particular `$setExpectedRows(1)` for empty tables can be done just as easy in jaspResults (e.g., before sending the json), but of course it can also be done in javascript if you prefer.

**Comment by boutinb** [comment]
But this fix can be already pushed I suppose. The fix for table can be done after.

**Comment by TimKDJ** [comment]
Yeah, it can be done separately.
